Makefile

```diff
@@ -62,6 +62,11 @@ manager: lint-full ## Build manager binary
 clusterctl: lint-full ## Build clusterctl binary
 	go build -o bin/clusterctl sigs.k8s.io/cluster-api/cmd/clusterctl
 
+bin/%-gen: ./vendor/k8s.io/code-generator/cmd/%-gen ## Build code-generator binaries
+	go build -o bin/client-gen ./vendor/k8s.io/code-generator/cmd/client-gen
```

Comment:
You can replace these 3 `go build` lines with this: `go build -o $@ ./$<`

Reply:
ooo, TIL

Reply:
Fixed, thanks!